wip

victorg1991 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 committed:
LPD-58672 Migrate to typescript

sandrodw3 committed:
LPD-58672 Allow sending a form id to affect only its inputs

# breaking

## What modules/apps/fragment/fragment-impl/src/main/resources/META-INF/resources/js/api/LocalizationSelect.tsx

Allow sending a specific form id to apply Localization Select events only to the inputs that belong to that form

## Why

We need a way to apply events only to the desired form for the new CMS side by side translation view

----

sandrodw3 committed:
LPD-58672 Add new configuration for localization management

sandrodw3 committed:
LPD-58672 Add localization options dropdown

sandrodw3 committed:
LPD-58672 buildLang

sandrodw3 committed:
LPD-58669 Isolate logic

sandrodw3 committed:
LPD-58669 Allow marking as translated and resetting translation

sandrodw3 committed:
LPD-58669 Explicitly remove translations when number is 0

sandrodw3 committed:
LPD-58669 Allow not creating translation input

sandrodw3 committed:
LPD-58669 Update usages

sandrodw3 committed:
LPD-58669 Don't create input when changing locale

sandrodw3 committed:
LPD-58669 Improve query

sandrodw3 committed:
LPD-58669 Adapt rich text fragment

sandrodw3 committed:
LPD-58669 Adapt file upload fragment

sandrodw3 committed:
LPD-58669 Adapt drag and drop upload fragment

sandrodw3 committed:
LPD-58669 We always want to show the dropzone is no file is selected

sandrodw3 committed:
LPD-58669 SF

sandrodw3 committed:
LPD-58669 Fix markup

sandrodw3 committed:
LPD-58669 Adapt video previewer fragment

sandrodw3 committed:
LPD-58669 buildLang

sandrodw3 committed:
LPD-58669 Adapt existing playwright tests

sandrodw3 committed:
LPD-58669 Rename

sandrodw3 committed:
LPD-58669 Move method to page

sandrodw3 committed:
LPD-58669 Fix name

sandrodw3 committed:
LPD-58669 Add new page for Localization Select fargment

sandrodw3 committed:
LPD-58669 Add new playwright tests

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
